src/bookshelf/rebar.config

```diff
@@ -4,7 +4,7 @@
 
 {erl_dep_retries, 10}.
 
-{require_otp_vsn, "18.*|19.*"}.
+{require_otp_vsn, "18.*|19.*|20.*"}.
```

Comment:
Should this be 20 only? Do we need to keep the older versions ?

Reply:
Same question for the other rebar.config files

Reply:
A question for @markan I guess.

Reply:
Lets leave this here for now - and delete it once we pick up a clean up ticket for older versions of erlang.